packages/gatsby/src/internal-plugins/internal-data-bridge/gatsby-node.js

```diff
@@ -129,6 +143,42 @@ exports.sourceNodes = ({ createContentDigest, actions, store }) => {
   })
 }
 
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions
+  const typeDefs = `
+    type Site implements Node {
+      buildTime: Date @dateformat
+    }
+  `
+  createTypes(typeDefs)
+}
+
+exports.createResolvers = ({ createResolvers }) => {
+  const resolvers = {
+    Site: {
+      buildTime: {
```

Comment:
Should we maybe deprecate this field and suggest querying for siteBuildMetadata directly if needed? @freiksenet

Reply:
I think that's one for v3

Reply:
I'll add a note about this in v3 wishlist

Reply:
This will defeat the purpose of that PR, cause we want nodes to stop changing.